mne_qt_browser/_pg_figure.py

```diff
@@ -4234,6 +4232,80 @@ def __init__(self, **kwargs):
             # disable histogram of epoch PTP amplitude
             del self.mne.keyboard_shortcuts["h"]
 
+        self.create_menus()
+
+    def create_menus(self):
+        """Creates and returns the application's menu bar with properly aligned shortcuts."""
+        if not self.menuBar():
+            self.setMenuBar(QMenuBar(self))
+        menu_bar = self.menuBar()
+        mne_python = menu_bar.addMenu("MNE-Python")
+        view_menu = menu_bar.addMenu("View")
+        help_menu = menu_bar.addMenu("Help")
+        scroll_menu = view_menu.addMenu("Scroll")
+
+        def add_menu_action(menu, description, shortcut):
+            widget = QWidget()
+            layout = QHBoxLayout(widget)
+            layout.setContentsMargins(10, 2, 10, 2)
```

Comment:
... this could probably be tweaked rather than using spaces below to fix the spacing.